File: BotnetAPI/BotnetAPI/Models/Bot.cs
namespace BotnetAPI.Models
{
    public class Bot
    {
        public int Id { get; set; }
        public string UserId { get; set; }
        public string IP { get; set; }
        public string Platform { get; set; }
        public string Status { get; set; }
        public string SSHName { get; set; }
    }
}

File: BotnetWebApp/botnetwebapp/src/components/Bot.js
import React from "react";

function Bot({ bot, deleteBot }) {
  const handleDelete = () => {
    deleteBot(bot);
  };

  return (
    <tr>
      <td>{bot.id}</td>
      <td>{bot.ip}</td>
      <td>{bot.platform}</td>
      <BotStatus bot={bot} />
      <td>
        <button onClick={handleDelete}>Delete?</button>
      </td>
    </tr>
  );
}

function BotStatus({ bot }) {
  if (bot.status === "Working") {
    return <td style={{ color: "green" }}>{bot.status}</td>;
  } else {
    return <td style={{ color: "red" }}>{bot.status}</td>;
  }
}

export default Bot;
